### ISSUE TITLE
Returning typeError: 'ABCMeta' object is not subscriptable

### ISSUE THREAD

**Issue by philmnoir** [juftin/lunchable-primelunch#5]
I have successfully installed PrimeLunch and it appeared to have accepted the API Token. First time running the command 'lunchable plugins primelunch -f ~/Downloads/amazon_order_history.csv', received the following error:

`Returning typeError: 'ABCMeta' object is not subscriptable`

**Comment by juftin** [comment]
Hey @philmnoir are you able to share some more stacktrace and details about what Python version you're running?

**Comment by juftin** [comment]
I think this is probably a Python 3.8 bug caused by some type hints behavior on Python <= `3.8`

There should be an easy fix by back-porting some type annotation behavior:
```
from __future__ import annotations
```

**Comment by juftin** [comment]
Yep, confirmed this is a Python 3.8 issue:

```console
❯ pipx run --python $(which python3.8) lunchable-primelunch -f ~/Downloads/amazon_order_history.csv -w 28
NOTE: running app 'primelunch' from 'lunchable-primelunch'
Traceback (most recent call last):
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/bin/primelunch", line 8, in <module>
    sys.exit(primelunch())
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/lib/python3.8/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/lib/python3.8/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/lib/python3.8/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/lib/python3.8/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/lib/python3.8/site-packages/lunchable_primelunch/cli.py", line 47, in primelunch
    from lunchable_primelunch.primelunch import PrimeLunch
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/lib/python3.8/site-packages/lunchable_primelunch/primelunch.py", line 29, in <module>
    class PrimeLunch(LunchableApp):
  File "/Users/justinflannery/.local/pipx/.cache/bb7f5ee1113edd4/lib/python3.8/site-packages/lunchable_primelunch/primelunch.py", line 36, in PrimeLunch
    file_path: str | os.PathLike[str] | pathlib.Path,
TypeError: 'ABCMeta' object is not subscriptable
```

**Comment by juftin** [comment]
Closed by https://github.com/juftin/lunchable-primelunch/commit/3fdbb276647606e4ff3250946d562d33a2895ff4

This was released on `lunchable-primelunch==1.0.3` https://github.com/juftin/lunchable-primelunch/releases/tag/v1.0.3. You will need to upgrade your installation of `lunchable-primelunch` to get this fix.
